packages/docs/content/getting-started/for-developers.mdx

```diff
@@ -0,0 +1,385 @@
+---
+title: For developers
+order: 10
+---
+
+import namingConventionsImg from '../../src/images/naming-convention.svg';
+import { Alert } from '@cmsgov/design-system';
+
+Greetings, Developers! This page will walk you through all the basics of using the design system, but another good way to get a jump start is to look through [our example projects on GitHub](https://github.com/CMSgov/design-system/tree/main/examples/).
+
+As you go through this documentation, keep in mind that the CMS Design System is a [family of design systems](/getting-started/child-design-systems/), and each of these design systems is distributed under a different package name:
+
+- [`design-system`](?theme=core) (Core Design System)
+- [`ds-healthcare-gov`](?theme=healthcare) (Healthcare.gov Child Design System)
+- [`ds-medicare-gov`](?theme=medicare) (Medicare.gov Child Design System)
+
+## Getting the design system assets
+
+How you get the design system assets depends on the needs of your project and your workstyle. You could...
+
+1. [Install the npm package](#option-1-install-using-npm)
+    - Good option if your project already uses [npm](https://www.npmjs.com/) for package management
+    - Makes it easy to get patch updates for your major/minor version
+2. [Reference files directly from the CDN](#option-2-reference-assets-from-the-cdn)
+    - Good option if your project doesn't use [npm](https://www.npmjs.com/)
+    - CDN files are versioned, so you always know what version you're using, and those files will be cached by the client's browser
+3. [Download the design system assets directly from GitHub](#option-3-download-assets-directly)
+    - This is not recommended, but it's available if no other options will work for your project
+
+### Option 1: Install using npm
+
+If you already have [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/) set up for your project, you can add it to your list of dependencies using one of the commands below. **Note that the following example will display a different package name depending on the theme you have selected.**
+
+<ThemeContent onlyThemes={['core']}>
+
+```sh
+npm install --save @cmsgov/design-system
+# or
+yarn add @cmsgov/design-system
+```
+
+</ThemeContent>
+
+<ThemeContent onlyThemes={['healthcare']}>
+
+```sh
+npm install --save @cmsgov/ds-healthcare-gov
+# or
+yarn add @cmsgov/ds-healthcare-gov
+```
+
+</ThemeContent>
+
+<ThemeContent onlyThemes={['medicare']}>
+
+```sh
+npm install --save @cmsgov/ds-medicare-gov
+# or
+yarn add @cmsgov/ds-medicare-gov
+```
+
+</ThemeContent>
+
+#### Package file structure
+
+The npm package includes:
+
+- Pre-compiled SCSS files
+- Compiled CSS files
+- Compiled React components (CommonJS and ES modules)
+- Typescript definition files
+- Fonts and images
```

Comment:
I'm considering getting rid of this to cut down on content and just lean on the directory diagram below. What do you think, @forrestbaer and @zarahzachz?